CRM/Utils/File.php

```diff
@@ -318,6 +318,12 @@ public static function createFakeFile($dir, $contents = 'delete me', $fileName =
    * @param bool $dieOnErrors
    */
   public static function sourceSQLFile($dsn, $fileName, $prefix = NULL, $isQueryString = FALSE, $dieOnErrors = TRUE) {
+    if (FALSE === file_get_contents($fileName)) {
```

Comment:
Good idea.

The problem raised by Jenkins occurs because `sourceSQLFile()` can (counter-intuitively) be used on strings in lieu of files. Line 321 should check `!$isQueryString` to determine whether `$fileName` is a SQL filename or a SQL string.